packages/buttons/src/elements/ToggleButton.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button, { IButtonProps } from './Button';
+
+interface IToggleButtonProps extends IButtonProps {
+  /**
+   * Determine if the button is pressed or not. Use `'mixed'` to indicate
+   * whether the toggle controls other elements which do not share the same
+   * value.
+   */
+  isPressed?: boolean | 'mixed';
```

Comment:
As this is a native attribute, I believe `boolean` is incorrect here. This may need to be a strings for all values: `'true' | 'false' | 'mixed'`

Could we reference the `ButtonHtmlProps` interface to get the matching values from `aria-pressed`?

Reply:
It's not a native attribute, it is a prop which is eventually mapped to a corresponding attribute. As the types here are just a subset, I felt it simplified the API and there wasn't an overwhelming need to support the corresponding boolean strings.

<img width="784" alt="Screen Shot 2020-06-01 at 9 59 03 AM" src="https://user-images.githubusercontent.com/143773/83434121-ba380980-a3ee-11ea-88e6-f90b4b7f3ced.png">